src/Styles/fonts.scss

```diff
@@ -1,42 +1,36 @@
 @font-face {
-  font-family: 'Roboto';
-  font-style: normal;
-  font-weight: 200;
-  src: url('./Fonts/Roboto-Thin.ttf') format('ttf');
-}
-@font-face {
-  font-family: 'Roboto';
+  font-family: 'Rubik';
   font-style: normal;
   font-weight: 300;
-  src: url('./Fonts/Roboto-Light.ttf') format('ttf');
+  src: url('./Fonts/Rubik-Light.ttf') format('ttf');
 }
 @font-face {
-  font-family: 'Roboto';
+  font-family: 'Rubik';
   font-style: normal;
   font-weight: 400;
-  src: url('./Fonts/Roboto-Regular.ttf') format('ttf');
+  src: url('./Fonts/Rubik-Regular.ttf') format('ttf');
 }
 @font-face {
-  font-family: 'Roboto';
+  font-family: 'Rubik';
   font-style: normal;
   font-weight: 500;
-  src: url('./Fonts/Roboto-Medium.ttf') format('ttf');
+  src: url('./Fonts/Rubik-Medium.ttf') format('ttf');
 }
 @font-face {
-  font-family: 'Roboto';
-  font-style: normal;
-  font-weight: 600;
-  src: url('./Fonts/Roboto-Bold.ttf') format('ttf');
-}
-@font-face {
-  font-family: 'Roboto';
+  font-family: 'Rubik';
   font-style: normal;
   font-weight: 700;
-  src: url('./Fonts/Roboto-Black.ttf') format('ttf');
+  src: url('./Fonts/Rubik-Bold.ttf') format('ttf');
 }
 @font-face {
-  font-family: 'RobotoMono';
+  font-family: 'Rubik';
   font-style: normal;
-  font-weight: 500;
-  src: url('./Fonts/RobotoMono-Medium.ttf') format('ttf');
+  font-weight: 900;
```

Comment:
This change seems incongruent. If we're maintaining a `normal` style, shouldn't we expect the weight to match? (i.e. 500, and not 900)?